Code/GraphMol/MolDraw2D/Wrap/rdMolDraw2D.cpp

```diff
@@ -105,6 +109,73 @@ void drawMoleculesHelper2(MolDraw2D &self, python::object pmols,
                           python::object highlight_atom_radii,
                           python::object pconfIds, python::object plegends) {
   rdk_auto_ptr<std::vector<ROMol *> > mols = pythonObjectToVect<ROMol *>(pmols);
+  unsigned int nThere = mols->size();
+  rdk_auto_ptr<std::vector<std::vector<int> > > highlightAtoms;
```

Comment:
There's "some" repeated code in here that could stand to be refactored at some point, but it's not like this is likely to be used in another function/method.